src/assets/custom/css/_sass/_single-training.scss

```diff
@@ -1,9 +1,30 @@
-.single-training__instructor-bio {
-  @include small-and-medium {
+.single-training__accordion {
+  @include small {
+    padding-bottom: 350px;
+  }
+  @include medium {
     padding-top: 50px;
   }
-
   @include large-and-extra-large {
+    padding-bottom: ($vertical-overlap * 2) + 50px;
     padding-top: 100px;
   }
-}
\ No newline at end of file
+  @include extra-large {
+    @include max-width--framed;
+  }
+  background-color: $navy--ultralight;
+}
+
+.single-training__instructor-bio {
+  @include small {
+    margin-top: -300px;
```

Comment:
This looks like it may be used to create an overlap? If so we should use the `$veretical-overlap` var